atlascope/core/job_types/nucleus_detection.py

```diff
@@ -63,7 +64,20 @@ def detect_nuclei(input_image):
         input_image,
     )
 
-    return additional_features.to_dict("records")
+    return (
+        additional_features.to_dict("records"),
+        np.vectorize(lambda x: 255 if x > 0 else 0)(im_nuclei_seg_mask).astype(np.uint8),
+    )
+
+
+def get_result_image(input_image, nucleus_mask):
```

Comment:
Can you include a docstring for this function? And if possible, Python type tags for the inputs and output?

IIUC, this creates a new image that overlays a nucleus mask using a green color on top of an existing image. Is that right?